internal/pkg/agent/install/svc.go

```diff
@@ -106,3 +114,35 @@ const darwinLaunchdConfig = `<?xml version='1.0' encoding='UTF-8'?>
   </dict>
 </plist>
 `
+
+// A copy of the systemd config template from github.com/kardianos/service
+// with added .Config.Option.TimeoutStopSec option
```

Comment:
Nit: this comment is still referring to the first fix...

```suggestion
// A copy of the systemd config template from github.com/kardianos/service
// with added .Config.Option.KillMode option
```

Reply:
Thanks for catching.  Fixed in ad152a41ef3ca16763d2f65cfea67b27e54864bd.